### ISSUE TITLE
Tools are Windows only

### ISSUE THREAD

**Issue by ExoPlant** [srnyx/modpack#3]
The tools in the /Tools folder are windows only, aka weeb only, please fix ok thx bye

**Comment by srnyx** [comment]
Yeah, you’ll have to use something like Wine until I convert them into bash. But, if you want, you can convert them for me :)
